http4s/src/main/scala/caliban/Http4sAdapter.scala

```diff
@@ -43,6 +61,12 @@ object Http4sAdapter {
           result   <- executeToJson(interpreter, query)
           response <- Ok(result)
         } yield response
+      case req @ GET -> Root =>
+        for {
+          query    <- Task(getGraphQLRequest(req.params)).absolve
```

Comment:
`Task.fromEither` is more straightforward.

Reply:
thanks, learning ZIO!